swift-sdk/Internal/IterableUtil.swift

```diff
@@ -78,10 +79,11 @@ enum IterableResult<T, E> {
         return { _ in
             if let customActionDelegate = customActionDelegate {
                 _ = customActionDelegate.handle(iterableCustomAction: context.action, inContext: context)
+                
                 return true
-            } else {
-                return false
             }
+            
+            return false
         }
```

Comment:
If you want to remove the else portion, I think this would be better with a `guard` statement instead of `if`.